CONTRIBUTING.md

```diff
@@ -65,6 +65,10 @@ Codespaces is new technology that allows you to use a container as your developm
 CMake version 3.13 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
+### Dotnet
+
+Required to get and execute test proxy(see /doc/TestProxy.md). [website](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet)
```

Comment:
This link shouldn't have `en-us`. The link verification fails because of it.
```suggestion
Required to get and execute test proxy(see /doc/TestProxy.md). [website](https://learn.microsoft.com/dotnet/core/tools/dotnet)
```

Also, the link isn't the most appropriate for finding where to get `dotnet` from.

Resolved in https://github.com/Azure/azure-sdk-for-cpp/pull/4273